lib/v0_bindings_extensions.rb

```diff
@@ -145,6 +130,38 @@ def summarize(user_uuid:)
     Api::V0::Bindings::GetHighlightsSummaryParameters
   ].each do |klass|
     klass.class_exec do
+
+      def get_highlights(user_id)
+        filters = to_hash.except(:page, :per_page, :sets)
+        filters[:user] = []
+
+        if (sets.blank? || sets.include?('user:me')) && user_id.present?
+          filters[:user].push(user_id)
+        end
+
+        if sets.present? && sets.include?('curated:openstax') && scope_id.present?
```

Comment:
sets can include user:me AND curated:openstax eh? this method has more than one big responsibility...it's building a user filter AND getting highlights.  so perhaps just the construction of the filter could be encapsulated in another method or class.

Reply:
this string "curated:openstax' should prob. be stored somewhere (on the Highlight model perhaps) as a const.  once place to store very important strings like this